app/scripts/components/common/mapbox/layer-legend.tsx

```diff
@@ -76,9 +78,23 @@ const LayerLegendSelf = styled.div`
   &.reveal-exit-active {
     transition: bottom 240ms ease-in-out, opacity 240ms ease-in-out;
   }
+`;
+
+const LayerLegendSelf = styled.div`
+  display: flex;
+  flex-flow: column nowrap;
+  width: 16rem;
+  border-bottom: 1px solid ${themeVal('color.base-100')};
 
   ${WidgetItemHeader} {
-    padding: ${variableGlsp(0.5, 0.75)};
+    padding: ${variableGlsp(0.25, 0.75)};
+  }
+
+  &:only-child {
+    ${WidgetItemHeader} {
+      padding: ${variableGlsp(0.5, 0.75)};
```

Comment:
```suggestion
      padding: ${variableGlsp(0.5)};
```

Reply:
Thank you for the review 🙇 I reflected all your commit suggestions in https://github.com/NASA-IMPACT/veda-ui/pull/547/commits/1ca17925b7dd2a641530663024e572ef505c0d30 !